### DISCUSSION TITLE
re-connect deleted PR branch on revival

### DISCUSSION THREAD

**Discussion by xerc** [community/community#7780] [Pull Requests]
https://github.com/matthiasmullie/minify/pull/380/commits
https://github.com/xerc/minify/commits/patch-1

**Comment by davorpa** [comment]
Detached PRs can be recovered using `git` commands over upstream "HEAD refs"

Use the following GIT commands to preserve all related commits history:
```shell
git fetch upstream pull/NNNN/head:BBBB
git switch BBBB
git push -u origin BBBB
```

whereas

- `NNN`: Pull request number
- `BBBB`: new branch name to create using last PR commit. **Name it as same as the deleted branch** linked to that PR
